app/src/main/java/ua/in/badparking/ui/activities/MainActivity.java

```diff
@@ -47,7 +47,6 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        setContentView(R.layout.activity_main);
```

Comment:
Because this activity is already annotated with @ContentView annotation, that doing actually the same